libs/core/src/components/pds-sortable-item/test/pds-sortable-item.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from '@stencil/core/testing';
```

Comment:
It can be removed, but I feel like at least one simple/default e2e test that uses an actual browser is better than nothing at all. I could definitely be wrong here though so let me know!

Reply:
There may not necessarily be a need for all components to have an e2e test file, since not every component will have a flow that needs to be checked.